docs/source/nbconvert_library.ipynb

```diff
@@ -86,7 +86,7 @@
    },
    "outputs": [],
    "source": [
-    "from IPython import nbformat\n",
+    "from jupyter import nbformat\n",
```

Comment:
Ah, ok. Meh, I can't actually easily test these examples at the moment because I have my global install of IPython still at 3.x and Jupyter 4.0 installed in a virtualenv. Let me see if I can set it up so I can do some better testing here...